README-multiple-instance.md

```diff
@@ -0,0 +1,72 @@
+Copyright (c) 2016 Intel Corporation 
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+Multi node run allows distributed testing. You can run multiple applications on the same machine or on a pool of machines. This run typically uses 3 scripts:
+ - run_multiple_instance.sh  - Runs multiple instances of the server client and mongodb on the same machine or across different machines.
+   	 -2 run modes - bare metal run (default run mode 0) and container run (run mode 1)
+
+ - multiple_instance_config.json - The input configuration file for multiple instances. Each instance requires these parameters:
+ 	  {
```

Comment:
Do you mind surrounding the JSON block with \`\`\` so it correctly renders in the markdown output.